.ghci.repl

```diff
@@ -25,3 +25,24 @@
 :seti -Wno-unsafe
 :seti -Wno-star-is-type
 :seti -Wno-missing-deriving-strategies
+
+-- Turn on some language extensions you use a lot
+:seti -XFlexibleContexts -XOverloadedStrings -XTypeApplications
+
+-- Break on errors
+:seti -fbreak-on-error
+
+-- Automatically show the code around breakpoints
+:set stop :list
+
+-- Use a cyan lambda as the prompt
+:set prompt "\ESC[1;36m\STXλ \ESC[m\STX"
```

Comment:
I know @robrix always tells me to put things like this in my `~/.ghci`, but I kinda like this in here (and you can still redefine at the user level if you want to).

Reply:
I think it's worth it. I'm not emotionally attached to the cyan lambda, but by default the prompt is really unhelpful, I think.